instruments/keithley/keithley580.py

```diff
@@ -108,11 +108,8 @@ def polarity(self):
         value = self.parse_status_word(self.get_status_word())['polarity']
         if value == '+':
             return Keithley580.Polarity.positive
-        elif value == '-':
-            return Keithley580.Polarity.negative
         else:
-            raise ValueError('Not a valid polarity returned from '
```

Comment:
This else statement is not reachable. If the polarity is not in '+' or '-', `get_status_word()` will raise an error message, thus, this else can never be reached. Removed.